src/Api/Ecommerce/Callback.php

```diff
@@ -59,19 +59,19 @@ public function call()
         }
 
         if (isset($this->postedData['currency'])) {
-            $response->currency = (int)$this->postedData['currency'];
+            $response->currency = (int) $this->postedData['currency'];
         }
 
         if (isset($this->postedData['type'])) {
             $response->type = $this->postedData['type'];
         }
 
         if (isset($this->postedData['embedded_window'])) {
-            $response->embeddedWindow = (bool)$this->postedData['embedded_window'];
+            $response->embeddedWindow = $this->postedData['embedded_window'];
```

Comment:
Looks like the property might have been incorrectly hinted:
`Property Altapay\Response\CallbackResponse::$embeddedWindow (bool) does not accept string.`